spec/lib/hubspot/company_spec.rb

```diff
@@ -49,6 +49,24 @@
     its(['num_associated_contacts']) { should eql '1' }
   end
 
+  describe ".update!" do
+    cassette "company_update_class"
+    let(:company) { Hubspot::Company.create!("New Company #{Time.now.to_i}") }
```

Comment:
What do you think of avoiding the use of `its`, since `its` is deprecated in RSpec3, and the use of `lets`?

thoughtbot has a pretty strong opinion about avoiding the use of `let` :) 
There's a robots blog post that discusses why we avoid using `let`: https://robots.thoughtbot.com/lets-not

Alternative:
```
describe ".update!" do
 it "updates the record" do
  VCR.use_cassette("company_update_class") do
     .. test setup ..
     Hubspot::Company.update!(company.vid, params)
     .. expectations ..
   end
 end
end
```
The [DealPipeline spec](https://github.com/adimichele/hubspot-ruby/blob/009520d38b3102912c2d37a05dd2d8b5d4229c9f/spec/lib/hubspot/deal_pipeline_spec.rb#L8) follows a similar pattern.